CONTRIBUTING.md

```diff
@@ -1,80 +1,233 @@
-+ Community consent on proposed change. If it’s hard to achieve add to agenda of
-the next WG meeting..
+  It's hard to know what the future brings, so whenever possible decisions
+  should be made which allow for more options in the future. Sometimes this is
+  unintuitive: spec rules often begin more strict than necessary with a future
+  option to loosen when motivated by a real use case.
 
-**Steps**: Merge PR into “master” branch.
+* **Understandability is just as important as correctness**
+  The GraphQL spec, despite describing technical behavior, is intended to be
+  read by people. Use natural tone and include motivation and examples.
 
-**Post-Acceptance Changes Expected**: None.
+
+## RFC Contribution Champions
+
+Contributing to GraphQL requires a lot of dedicated work. To set clear
+expectations and provide accountability, each proposed RFC (request for
+comments) must have a *champion* who is responsible for addressing feedback and
+completing next steps. An RFC may have multiple *champions*. The spec editors
+are not responsible for completing RFCs which lack a *champion* (though an
+editor may be a *champion* for an RFC).
+
+An RFC which does not have a *champion* may not progress through stages, and can
+become stale. Stale proposals may be picked up by a new *champion* or may
+be *rejected*.
+
+
+## RFC Contribution Stages
+
+RFCs are guided by a *champion* through a series of stages: *strawman*,
+*proposal*, *draft*, and *accepted* (or *rejected*), each of which has suggested
+entrance criteria and next steps detailed below. RFCs typically advance one
+stage at a time, however may advance multiple stages at a time. Stage
+advancements typically occur during
+[Working Group](https://github.com/graphql/graphql-wg) meetings, however may
+occur online.
+
+All RFCs start as either a *strawman* or *proposal*.
+
+## Stage 0: *Strawman*
+
+The *strawman* stage captures described problems or partially-considered
+solutions which lack the entrance criteria to be considered a *proposal*. A
+*strawman* may be an issue or a pull request (though an illustrative pull
+request is preferrable).
+
+*There is no entrance criteria for a Strawman*
+
+As implied by the name [strawman](https://en.wikipedia.org/wiki/Straw_man_proposal),
+the goal at this stage is to knock it down (*reject*) by considering other
+possible related solutions, showing that the motivating problem can be solved
+with no change to the specification, or that it is not aligned with the
+*guiding principles*.
+
+Once determined that the *strawman* is compelling, it should seek the entrance
+criteria for *proposal*.
+
+
+## Stage 1: *Proposal*
+
+An RFC at the *proposal* stage is a solution to a problem with enough fidelity
+to be discussed in detail, backed by a willing *champion*. A *proposal*'s goal
+is to make a compelling case for acceptance by describing both the problem and
+the solution via examples and spec edits. A *proposal* should be a pull request.
```

Comment:
Thesis: discussion stage
Requirements: can be discussed in detail, backed by a champion
Why: to make a case for both the problem and potential solutions. It should foster discussion about alternatives.
```
An RFC at the *proposal* stage is a solution to a problem with enough fidelity
to be discussed in detail. It must be backed by a willing *champion*. 
A *proposal*'s goal is to make a compelling case for acceptance by describing
both the problem and the solution via examples and spec edits. A *proposal* 
should be a pull request.
```